jenkins/run-tests.sh

```diff
@@ -40,6 +40,7 @@ pip install -U pip
 pip install -r requirements.txt
 pip install coverage[toml]
 pip install -e .  # use source files for coverage report
+playwright install-deps
 playwright install
```

Comment:
```suggestion
playwright install --with-deps chromium
```
might be better if we only use the chromium browser for testing